test/dynamo/test_graph_deduplication.py

```diff
@@ -97,19 +97,19 @@ class GraphModule(torch.nn.Module):
     def forward(self, primals_1: "f32[10, 10]", primals_2: "f32[10, 20]"):
         sin: "f32[10, 20]" = torch.ops.aten.sin.default(primals_2)
 
-        ___forward_subgraph_0_post_graph = self.___forward_subgraph_0_post_graph
-        invoke_subgraph_5 = torch.ops.higher_order.invoke_subgraph(___forward_subgraph_0_post_graph, '___forward_subgraph_0_post_graph', (primals_1, sin));  ___forward_subgraph_0_post_graph = sin = None
+        ___forward_subgraph_0_0_post_graph = self.___forward_subgraph_0_0_post_graph
```

Comment:
what is the additional 0 in the name now?

Reply:
I see, so forward subgraphs always have _0 in the second number?

Reply:
This is something that I want to fix/simplify in a future PR. But the current pattern is

First 0 - inserted by Dynamo to distinguish between different subgraphs
Second 0 - inserted by the AOTDIspatcher backward graph to distinguish between different bwd subgraphs (the purpose of this PR).